src/middleware.ts

```
@@ -1,28 +1,55 @@
+        Authorization: `Bearer ${accessToken}`,
+      },
+    })
+
+    const result = await res.json()
+
+    if (result.result === 'SUCCESS') {
+      isAuthenticated = true
+    } else {
+      shouldRemoveCookies = true
+    }
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)